tests/test_generate.py

```diff
@@ -1,69 +1,56 @@
 import unittest
+
```

Comment:
nit: remove unnecessary empty lines

Reply:
This is auto-formatted by yapf according to the specs in this repo